articles/101_build_tool.md

```diff
@@ -210,9 +202,15 @@ The tool performs an "isolated" build like `catkin_make_isolated` and `catkin_to
 - Python package with a `package.xml` file.
 - Python package without a manifest file (extracting the package name and dependencies from the `setup.py` file).
 
+### colcon
+
+When the first draft of this article was written the conclusion was to not to spend any resources towards a universal build tool.
+As a consequence the author of this article went ahead and developed [colcon](https://github.com/colcon/) as a personal project.
```

Comment:
While it is interesting for me to see how decisions are made at OSRF, maybe this part could still just state the relevant technical facts, such as: "The discussion about the first draft was inconclusive. Colcon was created as a PoC to demonstrate the viability of the approach of a unified build tool layer above respective build-system libraries."

Reply:
Maybe replace "As a consequence" with "Meanwhile", to reduce the risk of sounding accusatory.

Reply:
> Maybe replace "As a consequence" with "Meanwhile", ...

Updated in d326e110396d2640aad629d41cfa713598cf0cb5.